bugbug/tools/code_review.py

```diff
@@ -628,6 +628,39 @@ def parse_text_for_dict(text):
     return file_content
 
 
+# Find length of the common path
+def len_common_path(f1, f2):
+    f1_subsystems = f1.split("/")
+    f2_subsystems = f2.split("/")
+
+    idx_diff = [
+        idx
+        for idx, (sub1, sub2) in enumerate(zip(f1_subsystems, f2_subsystems))
+        if sub1 != sub2
+    ]
+    max_common_path_length = idx_diff[0]
```

Comment:
No it should be 0 because it's the idea of the first mismatches subsystem, going through the subsystem from the root to the furdest part right in the subsystem path. But I will apply the other suggestion (using next).